src/plots/piePlot.ts

```diff
@@ -355,8 +355,15 @@ export module Plots {
           { x: x + measurement.width, y: y + measurement.height }
         ];
 
-        var sliceIndices = corners.map((corner) => this._sliceIndexForPoint(corner));
-        var showLabel = sliceIndices.every((index) => index === datumIndex);
+        var absoluteCenter = { x: this.width() / 2, y: this.height() / 2 };
```

Comment:
Remember to put the github comment below the relevant code.  Putting it above will hide the relevant reviewed block.